### PR TITLE
15- aspect j method logging

### PR DESCRIPTION
**Pull request by ivshapovalov**
Connect #15 

- remove almost all logs
- add aspects for method before and after logging, exception handle logging
- rename endpoint urls for every microservice
- add aspect logger unit tests
- add checking of team members, returning from teams microservice

### CODE REVIEW

File: src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+        Object actual = debugAroundExceptionLogger.logExceptionHandleMethods(proceedingJoinPoint);
+
+        verify(proceedingJoinPoint).getArgs();
+        verify(proceedingJoinPoint).toShortString();
+        verify(proceedingJoinPoint).proceed();
+        addLogLineToLogs(warnMessage, message.replace("execution", "exception"),
+                exception.detailMessage());
+        verifyNoMoreInteractions(proceedingJoinPoint);
+        assertThat(actual, is(expected));
```
Comment (OlgaKulikova) [inline src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java]: Just want to clarify - call.proceed() has to return new empty Object()?

Comment (ivshapovalov) [reply src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java]: In production call.proceed() return result of original wrapped method execution. In test it returns as we wish.

Comment (OlgaKulikova) [reply src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java]: Сомнительный assert)) проверяем работу мока. Но предложить пока ничего не могу, потому пусть будет так.

**Review by OlgaKulikova — DISMISSED** [review]

File: src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java (hunk 1 above)
Comment (ivshapovalov) [reply src/test/java/ua/com/juja/microservices/teams/slackbot/aspect/DebugAroundExceptionLoggerTest.java]: Sure. Asserts removed. PR has been updated

**Review by OlgaKulikova — APPROVED** [review]

**Pull request merged**